features/user_can_delete_message.feature

```diff
@@ -0,0 +1,35 @@
+Feature: User can empty inbox
+  As a user
+  In order to clean up my inbox
+  I want to be able to delete messages
+
+  Background:
+    Given the following user exist:
+
+      | name   | email          | password  | id |
+      | dmybob | dmybob@bob.bob | bobbobbob | 1  |
```

Comment:
please make the users more descriptive
e.g
user_1@mail.com, password etc.

Reply:
ok